src/arraytraits.rs

```diff
@@ -6,17 +6,17 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use std::hash;
-use std::iter::FromIterator;
+use alloc::boxed::Box;
+use alloc::vec::Vec;
 use std::iter::IntoIterator;
 use std::mem;
 use std::ops::{Index, IndexMut};
-use alloc::boxed::Box;
-use alloc::vec::Vec;
+use std::{hash, mem::size_of};
+use std::{iter::FromIterator, slice};
 
-use crate::imp_prelude::*;
 use crate::iter::{Iter, IterMut};
 use crate::NdIndex;
+use crate::{dimension, imp_prelude::*};
```

Comment:
I don't know what tool put this use here, but prelude-like imports should be on their own line, with a newline separating from the others.